packages/fxa-auth-server/config/index.ts

```diff
@@ -1866,7 +1864,7 @@ if (conf.get('oauthServer.openid.keyFile')) {
   // If the file doesnt exist, or contains an empty object, then there's no active key.
   conf.set('oauthServer.openid.key', null);
   if (fs.existsSync(keyFile)) {
-    const key = JSON.parse(fs.readFileSync(keyFile));
+    const key = JSON.parse(fs.readFileSync(keyFile) as any);
```

Comment:
Technically readFileSync can return non-string data, though it shouldn't in this case, thus the any type.

Reply:
Because it could also come out as a Buffer, which doesn't like doing a direct cast to string, so then it would have to be something like:
`(fs.readFileSync(keyFile) as unknown) as string`

Reply:
I think if we impose an encoding, this always results in a string - e.g. `fs.readFileSync(keyFile, 'utf-8')`